ui/components/MesheryPatterns.js

```diff
@@ -485,7 +485,7 @@ function MesheryPatterns({
         body : pattern_file,
       }, () => {
         updateProgress({ showProgress : false });
-        enqueueSnackbar("Pattern Successfully Deployed!", {
+        enqueueSnackbar("Pattern successfully deployed", {
```

Comment:
Update Pattern to Design

Reply:
Should I update Pattern to Design on every change or on this particular comment?